src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -359,6 +359,8 @@ public MetadataStringDecoder GetMetadataStringDecoder()
             return _metadataStringDecoder;
         }
 
+        // Overriding 'protected internal' virtuals requires a specific signature if overriding from within
```

Comment:
Removed the comment. The CoreRT code and the bridge code are in the same assembly now.